ext/pdo_sqlite/sqlite_driver.c

```diff
@@ -28,6 +28,7 @@
 #include "zend_exceptions.h"
 #include "sqlite_driver_arginfo.h"
 
+
```

Comment:
No unrelated whitespace changes please

Reply:
fixed it, but leaving in 1 whitespace fix: https://github.com/php/php-src/pull/18320/files#diff-f74636a1b0487c077f6a68102ecdd5484e4e14a7159a7f9226f5bd6b3cfdcb10R757

on line 757 there it used 4 spaces while the surrounding code use 1x tab.

Reply:
Okay that's fine